Matlab/syntax_test_punctuation.m

```diff
@@ -0,0 +1,21 @@
+% SYNTAX TEST "Packages/Matlab/Matlab.sublime-syntax"
+
+
+x = [ 1.76 ]
+// <- source.matlab punctuation.definition.brackets.begin.matlab
```

Comment:
These assertion lines should start with `%` since you used that before `SYNTAX TEST` on the first line.